src/spectral/commands.ts

```diff
@@ -45,7 +59,7 @@ export const scanWorkSpaceFolders = async ({
     text: `$(loading~spin)`,
     tooltip: 'Spectral is scanning',
   })
-  await contextService.setContext(SCAN_STATE, ScanState.inProgress)
+  contextService.setContext(SCAN_STATE, ScanState.inProgress)
```

Comment:
why removing the await?

Reply:
not needed, probably added by mistake

Reply:
right I didn't notice that you removed the async from the functions.